lib/flutter_festival_home_page.dart

```diff
@@ -1,18 +1,19 @@
 import 'package:flutter/material.dart';
-import 'package:hackathon_flutterturkiye_org/widgets/footer_view.dart';
-import 'package:hackathon_flutterturkiye_org/widgets/section_title.dart';
-import 'package:hackathon_flutterturkiye_org/widgets/speakers_section.dart';
+import 'package:hackathon_flutterturkiye_org/core/ui/section_title.dart';
+import 'package:hackathon_flutterturkiye_org/footer/ui/footer_view.dart';
+import 'package:hackathon_flutterturkiye_org/speaker/ui/speakers_section.dart';
 
-class HomePage extends StatefulWidget {
-  HomePage({Key key, this.title}) : super(key: key);
+class FlutterFestivalHomePage extends StatefulWidget {
```

Comment:
Why we changed the name of Home Page? Will we add another page?

Reply:
Because it will be a single page web page :)

Reply:
If it is a single web page, Why, we changed the name of it? It can be HomePage.